pypowsybl/network.py

```diff
@@ -318,17 +323,22 @@ def write_single_line_diagram_svg(self, container_id: str, svg_file: PathOrStr)
         svg_file = _path_to_str(svg_file)
         _pp.write_single_line_diagram_svg(self._handle, container_id, svg_file)
 
-    def get_single_line_diagram(self, container_id: str) -> Svg:
+    def get_single_line_diagram(self, container_id: str, use_name: bool = False, center_name: bool = False, diagonal_label: bool = False, topological_coloring: bool = True) -> Svg:
         """
         Create a single line diagram from a voltage level or a substation.
 
         Args:
             container_id: a voltage level id or a substation id
+            use_name: (layout parameter) use names instead of ids in labels
```

Comment:
Your documentation makes me think: maybe we could group those parameters in a small class `LayoutParameters` ?
(only on the python side, not all the way to java)

Maybe this would scale better if we add other parameters in the future, or other arguments to the method (like custom labels).

Reply:
that's a great suggestion!  done.